### ISSUE TITLE
Add Cedrik Fuoco as a committer

### ISSUE THREAD

**Issue by doug-walker** [AcademySoftwareFoundation/OpenColorIO#1735]
As discussed at the recent TSC meeting, I'm nominating Cedrik to be added to the Committers list.  Several others on the TSC have also previously mentioned this would be a great idea.  Cedrik has been the developer on the Autodesk-contributed features for OCIO 2.2 such as the config archiving feature and the built-in configs feature.  He has also contributed bug fixes and many improvements to the build system.  He is currently working on bug fixes for 2.2.1 and starting work on features for OCIO 2.3.  He also contributes to discussions on Slack.

In addition, he has become the primary person integrating development work from the main branch into the released branches to support the OCIO patch releases, as needed.  Being a committer will ease this task in particular.

In summary, Cedrik has demonstrated his in-depth knowledge of many areas of the OCIO code-base through his many contributions and I believe he would be an excellent and very helpful addition to our list of committers.

**Comment by mjtitchener-fn** [comment]
Sounds good to me!

**Comment by carolalynn22** [comment]
100% Agree! Thank you for all the contributions and hard work, Cedrik!

**Comment by remia** [comment]
Strongly agree, thank you for your contributions Cedrik!

**Comment by cedrik-fuoco-adsk** [comment]
Thank you!

**Comment by doug-walker** [comment]
Cedrik has been added, thanks everyone for your support!
